packages/mesos/windows.build.ps1

```diff
@@ -1,184 +1,88 @@
 $ErrorActionPreference = "stop"
-# Mesos configurationsG
-$MESOS_DIR = "c:\mesos-tmp"
-$MESOS_BUILD_DIR = Join-Path $MESOS_DIR "build"
-$MESOS_GIT_REPO_DIR = Join-Path $MESOS_DIR "mesos"
-$MESOS_BUILD_OUT_DIR = Join-Path $MESOS_DIR "build-output"
 
-function New-Directory {
-    Param(
-        [Parameter(Mandatory=$true)]
-        [string]$Path,
-        [Parameter(Mandatory=$false)]
-        [switch]$RemoveExisting
-    )
-    if(Test-Path $Path) {
-        if($RemoveExisting) {
-            # Remove if it already exist
-            Remove-Item -Recurse -Force $Path
-        } else {
-            return
-        }
+$PKG_DIR = "c:\pkg"
+
+function Install-OpenSSL {
+    Write-Output "installing openssl"
+    # The argument options can be checked by running the installer
+    # with `/help`.
+    $p = Start-Process `
+        -FilePath $PKG_DIR/src/openssl/Win64OpenSSL-1_1_1e.exe `
+        -ArgumentList @("/VERYSILENT") `
+        -NoNewWindow `
+        -Wait `
+        -PassThru
+    if ($p.ExitCode -ne 0) {
+        Throw "failed to install OpenSSL"
     }
-    return (New-Item -ItemType Directory -Path $Path)
 }
 
-function New-Environment {
-    Write-Output "Creating new tests environment"
-    New-Directory $MESOS_BUILD_DIR
-    New-Directory $MESOS_BUILD_OUT_DIR -RemoveExisting
-    Write-Output "New tests environment was successfully created"
-}
+function Patch-Mesos {
+    Write-Output "patching mesos"
+    Push-Location $PKG_DIR/src/mesos
 
-function Start-ExternalCommand {
-    <#
-    .SYNOPSIS
-    Helper function to execute a script block and throw an exception in case of error.
-    .PARAMETER ScriptBlock
-    Script block to execute
-    .PARAMETER ArgumentList
-    A list of parameters to pass to Invoke-Command
-    .PARAMETER ErrorMessage
-    Optional error message. This will become part of the exception message we throw in case of an error.
-    #>
-    [CmdletBinding()]
-    param(
-        [Parameter(Mandatory=$true)]
-        [Alias("Command")]
-        [ScriptBlock]$ScriptBlock,
-        [array]$ArgumentList=@(),
-        [string]$ErrorMessage
-    )
-    PROCESS {
-        if($LASTEXITCODE){
-            # Leftover exit code. Some other process failed, and this
-            # function was called before it was resolved.
-            # There is no way to determine if the ScriptBlock contains
-            # a powershell commandlet or a native application. So we clear out
-            # the LASTEXITCODE variable before we execute. By this time, the value of
-            # the variable is not to be trusted for error detection anyway.
-            $LASTEXITCODE = ""
+    Get-ChildItem -Path $PKG_DIR/extra/windows-ee.patches -Filter "*.patch" | `

```

Comment:
Shouldn't the `windows-ee.patches` be renamed into `windows.patches`? They are not EE anymore.

Reply:
good catch, will rename to be consistent.